src/main.js

```diff
@@ -1,6 +1,10 @@
 import Vue from "vue";
 import App from "./App.vue";
 
+import BaseHeader from "./components/BaseHeader.vue";
+
+Vue.component("base-header", BaseHeader);
```

Comment:
zakaj si pa `BaseHeader` tkole "registriral" in ne v App komponenti?

Reply:
Tako sem jo registriral, ker sem razmišljal, da jo bom mogoče rabil ne samo v določeni komponenti, ampak v različnih  komponentah, globalno.

Reply:
ok... Kaj ti pa to omogoča več, kot če bi jo dal v App komponento na vrh strani?

Reply:
To, da jo lahko uporabljam v katerikoli komponenti, če jo dam pa v App komponento, jo pa lahko uporabljam samo v App komponenti